cluster/sbin/snappy-start-all.sh

```diff
@@ -80,5 +95,13 @@ export SKIP_CONF_COPY
 
 # Start Leads
 if [ "$clustermode" != "rowstore" ]; then
-  "$sbin"/snappy-leads.sh $CONF_DIR_ARG start
+  checkIfOkayToStartLeads
```

Comment:
empty parens will be good to add to indicate function call

Reply:
ok.

Reply:
Well this does not seem to work:
checkIfOkayToStartLeads()

If I put parenthesis then the finction is not called. What is wrong here. At other places also I checked for zero arg function just the name is mentioned.